src/CommerceContext.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace Frontkom\DrupalBehatDefinitions;
+
+use Behat\Gherkin\Node\TableNode;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+
+/**
+ * Class CommerceContext.
+ *
+ * Provide Behat step-definitions for common Commerce functionalities.
+ */
+class CommerceContext extends RawDrupalContext {
+
+  /**
+   * An array of promotions created during this context.
+   *
+   * @var array
+   */
+  protected $promotions = [];
```

Comment:
How about instead we just create an abstract base class, that cleans up something like "entities"?